app/assets/javascripts/application.js

```diff
@@ -1,8 +1,13 @@
+@import "bourbon";
+@import "neat";
+
+
+
```

Comment:
Nit: Two extra newlines. Please limit to one line of whitespace for readability.